src/main/scala/Server.scala

```diff
@@ -12,7 +12,7 @@ final class Server(
   }
 
   def training(turns: Int, map: Option[String] = None): Input = send {
-    Http.post(s"$endpoint/training").params(
+    Http.post(s"$endpoint/api/training").params(
```

Comment:
can you revert this bit, and replace this line https://github.com/MarcusHayward/vindinium-vishrugm/blob/master/src/main/scala/Kernel.scala#L77

with:
```
case (url, key) ⇒ Right(new Server(url.stripSuffix("/") + "/api", key))
```